app/src/pages/inside/common/itemInfo/attributesBlock/attributesBlock.scss

```diff
@@ -37,24 +38,41 @@
   }
 }
 .attribute {
-  display: inline-block;
+  display: inline-flex;
   font-family: $FONT-REGULAR;
   font-size: 12px;
   color: $COLOR--gray-60;
   text-decoration: none;
-  margin: 5px 10px 0 0;
+  margin: 5px 8px 0;
   word-break: break-all;
   word-wrap: break-word;
   line-height: 14px;
 
-  &:hover {
+  &:hover:not(.no-hover-effects) {
     color: $COLOR--charcoal-grey;
   }
 
-  &.cursor {
+  &.cursor:not(.no-hover-effects) {
     cursor: pointer;
   }
 
+  div {
+    max-width: 132px;
+    text-overflow: ellipsis;
+    overflow: hidden;
+    white-space: nowrap;
+  }
+
+  .value {
+    flex-basis: auto !important;
```

Comment:
What the reason using `!imporatnt`?
Can we avoid it?

Reply:
We need to change of current elements in this component. We can use `style` prop of element and pass flexBasis here, I'm not sure what aaproach is better :)